cf-custom-resources/lib/dns-cert-validator.js

```diff
@@ -78,43 +82,49 @@ let report = function (
   });
 };
 
+const aliasesChanged = function (
+    oldResourceProperties,
+    aliases,
+) {
+  if (!oldResourceProperties) {
+    return true;
+  }
+  let prevAliases = getAllAliases(
+      oldResourceProperties.Aliases
+  );
+  let aliasesToDelete = [...prevAliases,].filter(function (itm) {
+    return !aliases.has(itm);
+  });
+  let aliasesToAdd = [...aliases,].filter(function (itm) {
+    return !prevAliases.has(itm);
+  });
+  return aliasesToAdd.length + aliasesToDelete.length !== 0;
```

Comment:
im dumb why not just 
```suggestion
  return aliasesToAdd.length + aliasesToDelete.length;
```

Reply:
:homer-disappearing: